components/tidb_query/src/rpn_expr/impl_string.rs

```diff
@@ -1241,4 +1257,54 @@ mod tests {
             assert_eq!(output, expect_output);
         }
     }
+
+    #[test]
+    fn test_instr() {
+        let cases: Vec<(&str, &str, i64)> = vec![
+            ("a", "abcdefg", 1),
```

Comment:
Would you mind adding some invalid utf-8 char to these cases?

Reply:
Passing invalid utf-8 string to string maniplate function is undefined behaviour now(?)

https://github.com/tikv/tikv/pull/5946#pullrequestreview-318766329